### PR TITLE
[EL] Update `Rules` and `Official tournament support`

### PR DESCRIPTION
**Pull request by JohnVictoryz**
Updated the two outdated translations as shown in https://osu.wiki/status/el

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) ([off github](https://discord.com/channels/188630481301012481/218677502141399041/1165337633258475620))

### CODE REVIEW

File: wiki/Rules/el.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-outdated_translation: true
+outdated_translation: false
 ---
```
Comment (agatemosu) [inline wiki/Rules/el.md:3]: remove it, do not put "false"

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Rules/el.md (hunk 1 above)
Comment (JohnVictoryz) [reply wiki/Rules/el.md:3]: kk

Comment (JohnVictoryz) [reply wiki/Rules/el.md]: solved it :)

**Comment by agatemosu** [comment]
> * Refrain from continually updating the branch with `master` yourself. A [maintainer](https://osu.ppy.sh/wiki/People/osu%21_wiki_maintainers) will do this before merging anyway.

this line refers to that button, do not press that button

**Comment by JohnVictoryz** [comment]
> > * Refrain from continually updating the branch with `master` yourself. A [maintainer](https://osu.ppy.sh/wiki/People/osu%21_wiki_maintainers) will do this before merging anyway.
> 
> this line refers to that button, do not press that button

srry i pressed it now :((((

**Comment by cl8n** [comment]
has this been reviewed by another speaker of the language off-site?

**Comment by JohnVictoryz** [comment]
Yes on the discord server

**Comment by JohnVictoryz** [comment]
And here is the same link I send on discord for confirmation https://drive.google.com/file/d/1svgQkZtYAysVLJS3dodhyT3uzS0EhvGF/view?usp=drivesdk

**Comment by JohnVictoryz** [comment]
Also he misspelled something in the English version of the message srry but he basically said everything is okay

**Review by cl8n — APPROVED** [review]

**Pull request merged**